local-api/formlabs-api-local-openapi.yaml

```diff
@@ -1028,6 +1060,22 @@ paths:
                     - "FULL_BUILD_VOLUME"
                     - "FULL_PLATFORM_WIDTH"
                   default: "ZOOM_ON_MODELS"
+                yaw:
+                  type: number
+                  description: Yaw rotation in degrees for the camera's view, where 0º looks down the negative X-axis
+                  default: 45.0
+                pitch:
+                  type: number
+                  description: Pitch rotation in degrees for the camera's view, where 0º looks flat from the horizon and positive angles look down on models (in SLA scenes, toward the build platform)
+                  default: 35.264
```

Comment:
Should `roll` appear here?

Reply:
It's not in the thumbnail generation code (arguably it should be added) so no